### ISSUE TITLE
Click etc. events

### ISSUE THREAD

**Issue by allxie** [crubier/react-graph-vis#5]
I need the ability to use events. I'm trying to turn my nodes into links with a click event. I'm struggling to figure out how to do this with react-graph-vis. I'm going to modify my cloned copy to get it done, but this either needs to exist or there should be better documentation on how to accomplish it.

In vanilla vis we use...

`var network = new vis.Network(container, data, options);

```
network.on("click", function (params) {

   //blah blah params.nodes[0] blah turn it into a link blah

});`
```

but with this method network isn't defined. I'm using react-graph-vis as a component like so:

`<Graph options={options} graph={localGraphData}/>`

Thank you.

**Comment by crubier** [comment]
Fixed by #12

**Comment by crubier** [comment]
Checkout the updated readme for more info on this
